docs/tutorial/chat2.md

```diff
@@ -2,7 +2,10 @@
 
 ## Background
 
-The `chat2` application is a basic command-line chat app using the [Waku v2 suite of protocols](https://specs.vac.dev/specs/waku/v2/waku-v2). It connects to a [fleet of test nodes](fleets.status.im) to provide end-to-end p2p chat capabilities. The Waku team is currently using this application for internal testing. If you want try our protocols, or join the dogfooding fun, follow the instructions below.
+The `chat2` application is a basic command-line chat app using the [Waku v2 suite of protocols](https://specs.vac.dev/specs/waku/v2/waku-v2).
+It optionally connects to a [fleet of nodes](fleets.status.im) to provide end-to-end p2p chat capabilities.
+The Waku team is currently using this application on the `prod` fleet for internal testing.
```

Comment:
```suggestion
The Waku team is currently using this application on the production fleet `prod` for internal testing.
```

Reply:
I also suggest to mention to `wakuv2.prod` and `wakuv2.test` as the waku v2 fleet nodes in fleets.status.im

Reply:
Sure, added some clarifications in f8505c8